opentelemetry-exporter-cloud-trace/tests/test_cloud_trace_exporter.py

```diff
@@ -76,6 +77,23 @@ def setUp(self):
                 "int_key": AttributeValue(int_value=123),
             }
         )
+        self.agent_code = _format_attribute_value(
+            "opentelemetry-python {}; google-cloud-trace-exporter {}".format(
+                _strip_characters(
+                    pkg_resources.get_distribution("opentelemetry-sdk").version
+                ),
+                _strip_characters(cloud_trace_version),
+            )
+        )
+        self.example_trace_id = "6e0c63257de34c92bf9efcd03927272e"
+        self.example_span_id = "95bb5edabd45950f"
+        self.example_time_in_ns = 1589919268850900051
+        self.example_time_stamp = Timestamp(
+            seconds=1589919268, nanos=850900051
+        )
+        self.str_300 = "a" * 300
+        self.str_256 = "a" * 256
+        self.str_128 = "a" * 128
```

Comment:
If these are just immutable constants that don't need to be re-initialized for each test run, they can just be class constants IMO. Same goes for any of the above that can easily be expressed as class constants

Reply:
Resolved offline